TPP.Core/Setups.cs

```diff
@@ -45,6 +45,7 @@ public static ArgsParser SetUpArgsParser(IUserRepo userRepo, PokedexData pokedex
             argsParser.AddArgumentParser(new ManyOfParser(argsParser));
 
             argsParser.AddArgumentParser(new UserParser(userRepo));
+            argsParser.AddArgumentParser(new RoleParser());
```

Comment:
I failed to add comments to mark it, but I roughly divided this method into 3 sections of arguments parsers: "static" ones, self-referencial/recursive ones, and ones doing more fancy stuff with external dependencies. Can you push this line a few lines up, just below the pkmn species parser please?